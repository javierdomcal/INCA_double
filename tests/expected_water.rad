      0.000000000000E+00       0.000000000000E+00
      0.500000000000E-01       0.574174851760E+00
      0.100000000000E+00       0.184448160383E+01
      0.150000000000E+00       0.307408040231E+01
      0.200000000000E+00       0.394259775674E+01
      0.250000000000E+00       0.450203895171E+01
      0.300000000000E+00       0.495395398863E+01
      0.350000000000E+00       0.550288376583E+01
      0.400000000000E+00       0.626842258576E+01
      0.450000000000E+00       0.727173232705E+01
      0.500000000000E+00       0.846971643363E+01
      0.550000000000E+00       0.979373672692E+01
      0.600000000000E+00       0.111747305130E+02
      0.650000000000E+00       0.125547745841E+02
      0.700000000000E+00       0.138900324439E+02
      0.750000000000E+00       0.151494882687E+02
      0.800000000000E+00       0.163122225980E+02
      0.850000000000E+00       0.173646379983E+02
      0.900000000000E+00       0.182982432061E+02
      0.950000000000E+00       0.191081769248E+02
      0.100000000000E+01       0.197924106334E+02
      0.105000000000E+01       0.203514322295E+02
      0.110000000000E+01       0.207881616653E+02
      0.115000000000E+01       0.211078754126E+02
      0.120000000000E+01       0.213179932092E+02
      0.125000000000E+01       0.214276736307E+02
      0.130000000000E+01       0.214472445221E+02
      0.135000000000E+01       0.213875433986E+02
      0.140000000000E+01       0.212592589997E+02
      0.145000000000E+01       0.210723555707E+02
      0.150000000000E+01       0.208356373572E+02
      0.155000000000E+01       0.205564825278E+02
      0.160000000000E+01       0.202407501789E+02
      0.165000000000E+01       0.198928443743E+02
      0.170000000000E+01       0.195159057094E+02
      0.175000000000E+01       0.191120927694E+02
      0.180000000000E+01       0.186829120106E+02
      0.185000000000E+01       0.182295544840E+02
      0.190000000000E+01       0.177532013507E+02
      0.195000000000E+01       0.172552673122E+02
      0.200000000000E+01       0.167375614686E+02
      0.200000000000E+01       0.167375614686E+02
      0.210000000000E+01       0.156523787271E+02
      0.220000000000E+01       0.145206743127E+02
      0.230000000000E+01       0.133690059607E+02
      0.240000000000E+01       0.122229706295E+02
      0.250000000000E+01       0.111038163930E+02
      0.260000000000E+01       0.100272019732E+02
      0.270000000000E+01       0.900371832567E+01
      0.280000000000E+01       0.804018814509E+01
      0.290000000000E+01       0.714091744701E+01
      0.300000000000E+01       0.630852076924E+01
      0.310000000000E+01       0.554431125557E+01
      0.320000000000E+01       0.484842065204E+01
      0.330000000000E+01       0.421983013984E+01
      0.340000000000E+01       0.365643103751E+01
      0.350000000000E+01       0.315515954544E+01
      0.360000000000E+01       0.271219635969E+01
      0.370000000000E+01       0.232319819922E+01
      0.380000000000E+01       0.198352688982E+01
      0.390000000000E+01       0.168845136028E+01
      0.400000000000E+01       0.143330997042E+01
      0.410000000000E+01       0.121363044560E+01
      0.420000000000E+01       0.102521098204E+01
      0.430000000000E+01       0.864169136630E+00
      0.440000000000E+01       0.726965846738E+00
      0.450000000000E+01       0.610411294569E+00
      0.460000000000E+01       0.511658103471E+00
      0.470000000000E+01       0.428186039492E+00
      0.480000000000E+01       0.357781262782E+00
      0.490000000000E+01       0.298512321351E+00
      0.500000000000E+01       0.248704484405E+00
      0.510000000000E+01       0.206913606541E+00
      0.520000000000E+01       0.171900425379E+00
      0.530000000000E+01       0.142605970183E+00
      0.540000000000E+01       0.118128567166E+00
      0.550000000000E+01       0.977027571275E-01
      0.560000000000E+01       0.806802922579E-01
      0.570000000000E+01       0.665132550688E-01
      0.580000000000E+01       0.547392460906E-01
      0.590000000000E+01       0.449685187361E-01
      0.600000000000E+01       0.368728970272E-01
