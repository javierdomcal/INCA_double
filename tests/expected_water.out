 *******Job Finished************
 -----------Computational time----------------
 Total CPU time   15.9096556    
 Grid points computing time  0.423514992    
 Primitive quartet loop time   15.4861078    
 CPU time for intracule integrations   3.24249268E-05
 ---Primitive quartet time analysis-----------
 Reading .dm2p file-->   4.84240651E-02
 1st primitive screening-->  -8.38748515E-02
 2nd primitive screening-->  0.155566543    
 Grid points loop-->   15.1661291    
 ---------------------------------------------
 ------Grid point + primitive quartet info----
 Original grid points        3300
 Symmetry reduced grid points        1830
 Total number of reduced grid points        1830
 ---------------------------------------------
 --------------Accuracy check-----------------
 Sum of all DM2prim terms=   33.703058128264622        23693.806767268470     
 Thresholds used for DM2prim   10.000000000000000        10.000000000000000     
 Threshold used for screenings(Tau)=   1.0000000000000000E-008
 Computed limit for the screenings   4.3290043290043294E-011
 ---------------------------------------------
 --------------RADIAL INTEGRAL----------------
 Number of centres           1
           1 :::   5.1890166973119616E-310   0.0000000000000000        0.0000000000000000     
 Weights for centres   2.0000000000000000     
 Alpha parameter   1.0000000000000000     
 Gauss-Legendre nodes          30
 Gauss-Lebedev nodes         110
 ---------------------------------------------
 ---------------Final result------------------
 Intrac_integration=   44.994430777396104     
 Radial_integral error=  -5.5692226038956960E-003
